airbyte-api-server/src/main/openapi/api.yaml

```diff
@@ -0,0 +1,723 @@
+openapi: 3.0.2
+x-stoplight:
+  id: lo4tdzk9jxjfa
+info:
+  title: Airbyte API
+  version: 1.0.0
+paths:
+  "/connections/{connectionId}/sync":
+    post:
+      tags:
+        - connection
+        - RPC
+      responses:
+        "200":
+          description: Successful operation
+        "403":
+          $ref: "#/components/responses/NotAuthorizedResponse"
+        "404":
+          $ref: "#/components/responses/NotFoundResponse"
+      operationId: syncConnection
+      summary: Trigger a manual sync of the connection
+    #      security:
+    #        - bearerAuth: [ ]
+    parameters:
+      - name: connectionId
+        schema:
+          format: uuid
+          type: string
+        in: path
+        required: true
+      - name: X-Endpoint-API-UserInfo
+        in: header
+        schema:
+          type: string
+        description: an authorization header
+        required: false
+  #      - name: Authorization
+  #        in: header
+  #        schema:
+  #          type: string
+  ##          format: jwt
+  #        description: an authorization header
+  #        required: true
+  ##        type: string
+  "/connections/{connectionId}/reset":
+    post:
+      tags:
+        - connection
+        - RPC
+      responses:
+        "200":
+          description: Successful operation
+        "403":
+          $ref: "#/components/responses/NotAuthorizedResponse"
+        "404":
+          $ref: "#/components/responses/NotFoundResponse"
+      operationId: resetConnection
+      summary: Reset the data for the connection.
+      description: Reset the data for the connection. Deletes data generated by the connection in the destination. Resets any cursors back to initial state.
+    parameters:
+      - name: connectionId
```

Comment:
we are making a conscious choice that we use camel case for all field names, right? multi-word resource names are the same as well?  i know it's nitpicky but these are conventions that once we set, will be with us forever 😄 .